### PR TITLE
Add edit functionality to data bags

### PR DESCRIPTION
**Pull request by dishanktiwari2501**
### :nut_and_bolt: Description: What code changed, and why?

- Edit functionality has been added for Data Bag Object (which includes the Data Bag Items as well)

### :chains: Related Resources

https://github.com/chef/automate/blob/7b01ac3597028889180250d81f6815c89b15bede/components/docs-chef-io/content/automate/infra_server_views.md

### :white_check_mark: Checklist

**All PRs** must tick these:

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [x] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

All PRs **from Progress employees** should tick these if appropriate:

- [x] Tests added/updated? (all new code needs new tests)
- [x] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### CODE REVIEW

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 1]
```diff
@@ -144,6 +190,64 @@ The Chef Infra Server lets you delete environments one at a time. Select **Delet
...
+
+{{< figure src="/images/automate/edit-data-bag-item.png" alt="Edit a Data Bag Item">}}
+
+Whereas, to delete a data bag item, select **Delete**. In the dialog box displayed, select **Delete** to delete the data bag item.
+
+{{< figure src="/images/automate/delete-data-bag-item.png" alt="Delete a Data Bag Item">}}
+
+#### Delete a Data Bag
+
+Chef Infra Server lets you delete the existing data bag one at a time. To delete a data bag, select the ellipses icon and then **Delete**, as illustrated below:
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_infra_server.md:247]: Tautology, an ellipsis has three dots. You didn't ask for it, but here's a great [article about ellipses](https://uxdesign.cc/dot-dot-dot-7ce6170bfc7f) in UIx. Ellipsis is singular, ellipses plural. 

As to what we should call it, I think the "three dots icon" is probably clearest. @IanMadd recently upgraded us to fontawesome 5, so you have access to the icon. Ask Ian for guidance.

**Review by kagarmoe — CHANGES_REQUESTED** [review]
There are a couple of plsces where you need to tighten up the organization and a side-rable on ellipsis. This is generally clear and easy to follow.

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 2]
```diff
@@ -144,6 +144,70 @@ The Chef Infra Server lets you delete environments one at a time. Select **Delet
...
+
+{{< figure src="/images/automate/create-data-bag-button.png" alt="Create Data Bag Button">}}
+
+Selecting the **Create Data Bag** button opens a dialog box. Enter the name and select **Create**.
+
+{{< figure src="/images/automate/create-data-bag-popup.png" alt="Create Data Bag Dialog Box">}}
+
+#### Create a Data Bag Item
+
+To create a data bag item, select the data bag created above and follow the steps given below:
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_infra_server.md:169]: Try to write each section as an independent part so users can solve a problem (in this case, "How do I create a Data Bag Item?") independently of other content.

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 3]
```diff
@@ -144,6 +144,70 @@ The Chef Infra Server lets you delete environments one at a time. Select **Delet
...
+
+Select **Create Item**, as shown below:
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_infra_server.md:171]: Remove 'as shown below' and describe the location of the button in the step or in the alt-text (don't do both. Alt-text ideally does not repeat content.

```suggestion
Select **Create Item**
```

**Review by kagarmoe — COMMENTED** [review]
Good changes. These comments are mostly on patterns to resolve style. Nice to see the right punctuation on unordered lists with full-sentence items.

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 4]
```diff
@@ -144,6 +144,70 @@ The Chef Infra Server lets you delete environments one at a time. Select **Delet
...
+
+{{< figure src="/images/automate/create-data-bag-item-button.png" alt="Create Data Bag Item Button">}}
+
+Selecting the **Create Item** button opens a dialog box. Enter the **Data Bag Item ID** and the details of the key value pairs of that particular item. Select **Create**.
+
+{{< figure src="/images/automate/create-data-bag-item-popup.png" alt="Create Data Bag Item Dialog Box">}}
+
+#### Search for an Item
+
+Use the **Search data bag items** bar to find an item from the list of data bag items. Entering the name of an item in the search bar returns data bag items matching your search criteria.
```
Comment (IanMadd) [inline components/docs-chef-io/content/automate/chef_infra_server.md:181]:
```suggestion
Use the **Search data bag items** bar to find an item from the list of data bag items. Entering the name of an item in the search bar returns data bag items that match your search criteria.
```

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 5]
```diff
@@ -144,6 +144,70 @@ The Chef Infra Server lets you delete environments one at a time. Select **Delet
...
+
+{{< figure src="/images/automate/edit-and-delete-data-bag-item.png" alt="Edit and Delete a Data Bag Item">}}
+
+To delete a data bag item, select **Delete**. In the dialog box displayed, select **Delete** to delete the data bag item.
+
+{{< figure src="/images/automate/delete-data-bag-item.png" alt="Delete a Data Bag Item">}}
+
+#### Delete a Data Bag
+
+Chef Infra Server lets you delete the existing data bag one at a time. To delete a data bag, select the ellipses icon and then **Delete**, as illustrated below:
```
Comment (IanMadd) [inline components/docs-chef-io/content/automate/chef_infra_server.md:207]: You might try this locally first to see how it looks, but the fontawesome shortcode will add the ellipses icon into the text. 

```suggestion
Chef Infra Server lets you delete the existing data bag one at a time. To delete a data bag, select the ellipses icon {{< fontawesome class="fas fa-ellipsis-h" >}} and then **Delete**, as illustrated below:
```

File: components/docs-chef-io/content/automate/chef_infra_server.md (hunk 2 above)
Comment (dishanktiwari2501) [reply components/docs-chef-io/content/automate/chef_infra_server.md:169]: Noted! Making the possible changes to docs.

**Review by kagarmoe — APPROVED** [review]
tenor-78749959

**Review by IanMadd — APPROVED** [review]
tenor-5902556

**Pull request closed without merging**